platform/framework/src/FrameworkLayer.js

```diff
@@ -62,7 +67,7 @@ define([
             loader = new BundleLoader($http, $log, openmct.legacyRegistry),
             resolver = new BundleResolver(
                 new ExtensionResolver(
-                    new ImplementationLoader({}),
+                    new ImplementationLoader(),
```

Comment:
I'm not sure how this breaks tests, but as you can see on this changed line, an empty `{}` object was being passed in, but `ImplementationLoader` expects it to be a function and expects to call it, so I don't see how this code could possibly work, and hence why it seems unused.

Any ideas?